### DISCUSSION TITLE
Mui Language Request (English

### DISCUSSION THREAD

**Discussion by byoricon** [babalae/better-genshin-impact#277] [General]
add English plss

**Comment by renjako** [comment]
agree, UI only in english is very appreciated

**Comment by Astaroth-68** [comment]
Please, this has the potential to expand, even with an automatic translation

**Comment by k59319** [comment]
Please add English translation

**Comment by Joao-ALD** [comment]
Please add english UI 🙏

**Comment by huiyadanli** [comment]
https://github.com/babalae/better-genshin-impact/pull/2709   

After merging this PR, the app's UI will be multilingual, but core functions still rely on the Chinese game UI.
